packages/manager/src/features/NodeBalancers/NodeBalancerDetail/NodeBalancerSettings.tsx

```diff
@@ -25,7 +25,7 @@ export const NodeBalancerSettings = () => {
   const id = Number(nodeBalancerId);
   const { data: nodebalancer } = useNodeBalancerQuery(id);
   const { data: attachedFirewallData } = useNodeBalancersFirewallsQuery(id);
-  const displayFirewallInfoText = Boolean(attachedFirewallData?.data?.length);
+  const displayFirewallInfoText = !Boolean(attachedFirewallData?.data?.length);
```

Comment:
Optional: I think this would be a bit more readable.

```suggestion
  const displayFirewallInfoText = attachedFirewallData?.results === 0;
```